packages/office-ui-fabric-react/src/components/GroupedList/GroupedList.base.tsx

```diff
@@ -203,6 +217,11 @@ export class GroupedListBase extends React.Component<IGroupedListProps, IGrouped
       return null;
     }
 
+    const finalListProps: IListProps = {
+      ...(listProps || {}),
```

Comment:
nit: if you spread an object whose value is null or undefined, still works.

doesn't work for spreading arrays.